frontend/locales/de/translation.json

```
@@ -126,7 +126,8 @@
     },
     "processing": {
       "head": "Verarbeiten",
-      "export": "Für Statistik exportieren",
+      "export_csv": "Für Statistik exportieren",
+      "export_xlxs": "Für Statistik exportieren (Excel)",
```

```suggestion
      "export csv": "Für Statistik exportieren",
      "export xlxs": "Für Statistik exportieren (Excel)",
```

The convention followed for translation keys so far was to not put any delimiters between words.